### PR TITLE
chore(app): fixing windows CI regression on 10.0-release

### PR DESCRIPTION
**Pull request by lmiller1990**
I broke windows CI https://github.com/cypress-io/cypress/pull/20818. I am fixing it here.

### CODE REVIEW

**Review by tgriesser — APPROVED** [review]
Looks like this might have already been fixed in 10.0-release

File: packages/launchpad/cypress/tasks/snapshotsScaffold.ts [hunk 1]
```diff
@@ -40,9 +40,9 @@ async function snapshotScaffoldedFiles (expectedScaffoldDir: string, filesToDiff
 function compareScaffoldedFiles (filesToDiff: FileToDiff[]): SnapshotScaffoldTestResult {
   for (const { actual, expected } of filesToDiff) {
     try {
-      const read = (f: string) => fs.readFileSync(f, 'utf-8')
-      const actualContent = read(actual).trim()
-      const expectedContent = read(expected).trim()
+      const read = (f: string) => fs.readFileSync(f, 'utf-8').trim().replaceAll('\r\n', '\n')
```
Comment (lmiller1990) [inline packages/launchpad/cypress/tasks/snapshotsScaffold.ts:43]: windows line-endings was messing with the diff. I wish (wonder? maybe there is?) if there's a more idiomatic way that this. I wish `fs.readFileSync(..., { normalizeLineEndings: "\n")` or something.

**Comment by lmiller1990** [comment]
Ok! https://github.com/cypress-io/cypress/pull/20914/commits/8b3f64d796e20fb822e573060bab7102f61c629e

**Comment by lmiller1990** [comment]
Alright I fixed what I set out to fix (launchpad e2e passes on windows), gonna call it there. There is still some app tests failing, but we can fix those in another PR.

**Pull request merged**